pennychat/views.py

```diff
@@ -16,8 +19,26 @@ class PennyChatViewSet(viewsets.ModelViewSet):
     serializer_class = PennyChatSerializer
 
     def perform_create(self, serializer):
-        # TODO add participants and make request user's UserProfile as the Organizer
-        serializer.save()
+        if not self.request.auth:
+            raise NotAuthenticated
+        chat = serializer.save()
+        profile = UserProfile.objects.get(user=self.request.user)
```

Comment:
this is a great example of "what on earth do we do?!" when there is more that one UserProfile. 🤔